packages/communication-react/review/communication-react.api.md

```diff
@@ -1879,7 +1879,16 @@ export const MessageStatusIndicator: (props: MessageStatusIndicatorProps) => JSX
 
 // @public
 export interface MessageStatusIndicatorProps {
+    // (undocumented)
+    message?: ChatMessage;
     messageThreadReadCount?: number;
+    // (undocumented)
+    readReceiptForEachSender?: {
```

Comment:
Can we pass in a tooltip string instead of this? readReceiptsForEachSender sounds like we are passing too much info for just one indicator